rswag-specs/lib/rswag/specs/configuration.rb

```diff
@@ -26,9 +26,11 @@ def swagger_docs
       end
 
       def swagger_dry_run
-        @swagger_dry_run ||= begin
-          @rspec_config.swagger_dry_run.nil? || @rspec_config.swagger_dry_run
+        return @swagger_dry_run if defined? @swagger_dry_run
+        if ENV.key?('SWAGGER_DRY_RUN')
+          @rspec_config.swagger_dry_run = ENV['SWAGGER_DRY_RUN'] == '1'
```

Comment:
In order to maintain a familiar API, WDYT about keeping the env var value as a boolean?

```suggestion
          @rspec_config.swagger_dry_run = ENV['SWAGGER_DRY_RUN'] == true
```

Reply:
Unfortunately, all ENV vars come as strings.